### DISCUSSION TITLE
What is zen mode?

### DISCUSSION THREAD

**Discussion by Astraevs-12** [WeblateOrg/weblate#7555] [Q&A] [answered]
I am translating weblate.
But I don't know what zen mode is.
and What is Stringsdict?

**Comment by Geeyun-JY3** [comment, marked as answer]
Search the docs then you would get the answers: 
- https://docs.weblate.org/en/latest/user/translating.html#zen-mode
- https://docs.weblate.org/en/latest/formats.html#stringsdict

**Reply by Astraevs-12** [reply]
thanks for letting me know!

**Reply by Astraevs-12** [reply]
Do you know why the name is zen mode?

**Reply by Geeyun-JY3** [reply]
Initial issue: #375

I think you should ask the search engine and dictionary.

- https://en.wikipedia.org/wiki/Zen
- https://www.merriam-webster.com/dictionary/Zen
- https://www.merriam-webster.com/dictionary/Zen-like
- https://www.google.com/search?q=zen+mode
